digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/util/AssetDataImporter.scala

```diff
@@ -328,6 +423,28 @@ class AssetDataImporter {
     }
   }
 
+  def importRoadAddressHistory(conversionDatabase: DatabaseDef, importDate: String): Unit = {
+
+    print(s"\n${DateTime.now()} - ")
+    println("Starting to import history data %s ".format(importDate))
+
+    val roadMaintainerElys = Seq(0, 1, 2, 3, 4, 8, 9, 10, 12, 14)
+
+    //Delete all the previous road history
+    OracleDatabase.withDynTransaction{
```

Comment:
should we drop all the roadaddress and lrm table content?

Reply:
well, for now it is just deleting the history with "END_DATE" not null, we could delete all but we would lose the "current road" and would have to reinsert it

Reply:
yea, thought we wanted it like that which is why i asked it on daily to be sure. Didn't run test, but code seems ok to me.